File: problems/1134/solution.py
class Solution:
    def isArmstrong(self, N: int) -> bool:
        nlen, n, ans = int(math.log10(N) + 1), N, 0
        
        while n:
            ans += (n % 10) ** nlen
            n //= 10
        
        return ans == N

File: problems/1134/solution.cpp
class Solution {
public:
    bool isArmstrong(int n) {
        int numLen = log10(n) + 1,
            ans = 0,
            N = n;
        
        while (N) {
            ans += pow(N % 10, numLen);
            N /= 10;
        }
        
        return ans == n;
    }
};